pkger/service.go

```diff
@@ -362,6 +362,21 @@ func (s *Service) InitStack(ctx context.Context, userID platform.ID, stCreate St
 		return Stack{}, err
 	}
 
+	// Reject use of server-side jsonnet with stack templates
+	for _, u := range stCreate.TemplateURLs {
+		// While things like '.%6Aonnet' evaluate to the default encoding (yaml), let's unescape and catch those too
+		decoded, err := url.QueryUnescape(u)
+		if err != nil {
+			msg := fmt.Sprintf("stack template from url[%q] had an issue", u)
```

Comment:
Again, do we want to return this error, or is that too much information for an attacker?